timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -94,31 +100,31 @@ def run(self):
         problem_strings = []
         output_strings = []
 
-        tags_applied[rule.get("file_name")] = 0
+        tags_applied[rule.get("title")] = 0
         try:
             sigma_rule_counter += 1
             tagged_events_counter = self.run_sigma_rule(
                 rule.get("search_query"),
-                rule.get("file_name"),
+                rule.get("title"),
                 tag_list=rule.get("tags"),
-                status_good=rule.get('ts_use_in_analyzer'),
             )
-            tags_applied[rule.get("file_name")] += tagged_events_counter
+            tags_applied[rule.get("title")] += tagged_events_counter
         except:  # pylint: disable=bare-except
+            error_msg = "* {0:s} {1:s}".format(
+                rule.get("title"), rule.get("id")
+            )
             logger.error(
-                "Problem with rule in file {0:s}: ".format(
-                    rule.get("file_name")
-                ),
+                error_msg,
                 exc_info=True,
             )
-            problem_strings.append("* {0:s}".format(rule.get("file_name")))
+            problem_strings.append(error_msg)
```

Comment:
Since each rule has their own `run()` instance there is only one problem in this list if it occurs. So why not return the `error_msg` directly?